modules/serverless-rn-1-29-0.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-29-0_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.29
+
+{ServerlessProductName} 1.29 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-29-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.8.
+* {ServerlessProductName} now uses Knative Eventing 1.8.
+* {ServerlessProductName} now uses Kourier 1.8.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.8.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.8.
+* The `kn func` CLI plug-in now uses `func` 1.10.
+
+* Beginning with {ServerlessProductName} 1.29, the different product versions are available as follows:
+** The latest release is available through the `stable` channel.
+** Releases older than the latest are available through the version-based channels.
++
+To use these, update the channel parameter in the subscription object YAML file from `stable` to the corresponding version-based channel, such as `stable-1.29`.
++
+This change allows you to receive updates not only for the latest release, but also for releases in the Maintenance phase.
++
+Additionally, you can lock the version of the Knative (`kn`) CLI. For details, see section "Installing the Knative CLI".
+
+* You can now create {ServerlessProductName} functions through DevConsole using {product-title} Pipelines.
```

Comment:
maybe "the developer console" or "the web console"?

Reply:
went with "developer console"